assets/cubyz/tools/chisel.zig.zon

```diff
@@ -0,0 +1,65 @@
+.{
+	.blockTags = .{.mineable, .choppable},
+	.disabled = .{
+		0, 0, 1, 1, 1,
+		0, 0, 0, 1, 1,
+		1, 0, 0, 1, 1,
+		1, 1, 1, 0, 1,
+		1, 1, 1, 1, 0,
+	},
+	.optional = .{
+		0, 0, 0, 0, 0,
```

Comment:
You can make one or two more slots optional, either make only the tip required or make it like the shover.

Reply:
still looks the same.